apps/user-office-frontend/src/components/user/PeopleTable.tsx

```diff
@@ -234,6 +235,13 @@ const PeopleTable: React.FC<PeopleTableProps> = (props) => {
   }
   const EmailIcon = (): JSX.Element => <Email />;
 
+  const handleChangeCoIToPi = (user: BasicUserDetails) => {
+    try {
+      props.onRemove?.(user);
+      props.setPi?.(user);
+    } catch (error) {}
+  };
+
```

Comment:
Should we not be handling the error rather than silencing it .We should also  handle the error states in  onRemove and setPi  functions.